client/components/notice/style.scss

```diff
@@ -310,3 +310,57 @@ a.notice__action {
 		}
 	}
 }
+
+.notice.is-reskinned {
+	background: var(--studio-gray-0);
+	color: inherit;
```

Comment:
Not sure about "inherit" here. It works but it might be best to set it explicitly 🤔 
same for icon [below](https://github.com/Automattic/wp-calypso/pull/75333/files#diff-cb8e8eca0c2990d3d8994dcbc991f202c49257238de2438ba6795c383cbdb298R350)